client/src/App.jsx

```diff
@@ -1,11 +1,23 @@
 import React, { Component } from 'react';
+import Axios from 'axios';
+import Homes from './Homes';
 
+class App extends Component {
```

Comment:
There is no need to call super if there is no props (I believe you can actually remove the constructor function and the compiler will create one for you at run-time).